server/jetstream_cluster.go

```diff
@@ -7793,7 +7793,7 @@ func (mset *stream) processClusteredInboundMsg(subject, reply string, hdr, msg [
 		if err == nil {
 			err = NewJSAccountResourcesExceededError()
 		}
-		s.RateLimitWarnf(err.Error())
+		s.RateLimitWarnf("JetStream account limits exceeded for '%s': %s", jsa.acc().Name, err.Error())
```

Comment:
Would use GetName(), Does locking and also handles nil properly.

Reply:
Done.